samples/samples/async_client_sample.py

```diff
@@ -0,0 +1,648 @@
+     response = await async_client.read(request = request)
+
+     """Prints the response."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+# [END async client read]
+
+
+# [START async client execute_sql]
+async def execute_sql(database_object, async_client, task_num):
+     print("\nstarting task: ", task_num)
+
+     # Create the session object associated with the database object
+     session_object = session.Session(database_object)
+
+     """Create the session."""
+     session_object.create()
+
+     """Request object for execute sql."""
+     request = ExecuteSqlRequest(
+         session = session_object.name,
+         sql =  "SELECT SingerId, FirstName, LastName FROM Singers",
+     )
+
+     """Execute the sql statement."""
+     response = await async_client.execute_sql(request = request)
+
+     """Prints the response."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client execute_sql]
+
+
+# [START async client delete_session]
+async def delete_session(session_name, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     """Request object for delete session."""
+     request = DeleteSessionRequest(
+         name=session_name,
+     )
+
+     """Delete the session."""
+     await async_client.delete_session(request = request)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client delete_session]
+
+# [START async client list_sessions]
+async def list_sessions(database_object, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     """Request object for list session."""
+     request = ListSessionsRequest(
+         database = database_object.name,
+     )
+
+     """Get the list of sessions."""
+     response = await async_client.list_sessions(request = request)
+
+     """Prints the session."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client get_session]
+
+
+# [START async client get_session]
+async def get_session(database_object, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     # Create the session object associated with the database object
+     session_object = session.Session(database_object)
+
+     """Create the session."""
+     session_object.create()
+
+     """Request object for get session."""
+     request = GetSessionRequest(
+         name = session_object.name,
+     )
+
+     """Get the session."""
+     response = await async_client.get_session(request = request)
+
+     """Prints the session got."""
+     print(response)
+
+     print("\nEnding task: ", task_num)
+
+# [END async client get_session]
+
+
+# [START async client batch_create_session]
+async def batch_create_session(database_object, async_client, task_num):
+
+     print("\nStarting task: ", task_num)
+
+     """Request object for batch create session."""
+     request = BatchCreateSessionsRequest(
+         database = database_object.name,
+         session_count=1420,
```

Comment:
How did we arrive at this session count here? Or are we setting this to a random value?

Reply:
We are setting it to a random value. This parameter is the number of sessions to be created in this batch call, hence we can set any value to it.